src/__tests__/UserMenu.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import {mount, shallow} from 'enzyme';
+import React from 'react';
+import {UserMenu} from '../Header/UserMenu';
+
+describe('<UserMenu>', () => {
+  it('should add custom class name', () => {
+    const handleLogout = jest.fn();
+    const wrapper = shallow(
+      <UserMenu
```

Comment:
You have same wrapper in all test cases so it make sense to add a function that will return component object